spec/schema_tools/hash_spec.rb

```diff
@@ -16,7 +16,15 @@ class Contact
 end
 
 class OneOfDefinition
-  attr_accessor :person
+  attr_accessor :person, :one_of_schema, :all_of_schema, :any_of_schema
+end
+
+class AnyOfDefinition
+  attr_accessor :person, :one_of_schema, :all_of_schema, :any_of_schema
```

Comment:
I see, I accept your point. I don't think I fully understand our reasoning for using this gem. Anyhow, let's crack on.